package.json

```diff
@@ -88,19 +88,24 @@
     "@skypack/package-check": "^0.2.2",
     "@types/autoprefixer": "^9.7.2",
     "@types/copy-webpack-plugin": "^6.4.0",
+    "@types/cssnano": "^4.0.0",
     "@types/express": "^4.17.9",
     "@types/fs-extra": "^9.0.1",
     "@types/globby": "^9.1.0",
+    "@types/http-proxy-middleware": "^1.0.0",
     "@types/imagemin": "^7.0.0",
     "@types/jest": "^26.0.20",
+    "@types/json-stringify-safe": "^5.0.0",
     "@types/lodash": "^4.14.161",
-    "@types/node": "^14.14.31",
+    "@types/mini-css-extract-plugin": "^1.2.2",
+    "@types/node": "^14.14.37",
     "@types/node-notifier": "^8.0.0",
+    "@types/optimize-css-assets-webpack-plugin": "^5.0.2",
     "@types/pino": "^6.3.4",
     "@types/react": "^17.0.0",
     "@types/shell-quote": "^1.7.0",
-    "@types/webpack": "^4.41.26",
-    "@types/webpack-hot-middleware": "^2.25.3",
+    "@types/signale": "^1.4.1",
+    "@types/webpack-env": "^1.16.0",
```

Comment:
I moved all these devDependencies (typings) to root to make it easier to upgrade. I think I like them all up here better anyway, though. Thinking that they will not be moved back.